src/Files.App (Package)/Package.appxmanifest

```diff
@@ -119,12 +119,14 @@
                 </uap3:Extension>
 
                 <uap:Extension Category="windows.protocol">
-                    <uap:Protocol ReturnResults="none" Name="files-uwp" />
+                    <!--  This value changes based on the build branch (files, files-pre, files-dev).  -->
+                    <uap:Protocol ReturnResults="none" Name="files-dev" />
                 </uap:Extension>
 
                 <uap5:Extension Category="windows.appExecutionAlias">
                     <uap5:AppExecutionAlias>
-                        <uap5:ExecutionAlias Alias="files.exe" />
+                        <!--  This value changes based on the build branch (files, files-pre, files-dev).  -->
```

Comment:
This will break any existing scripts or shortcuts targeting `files.exe`.

Reply:
Yeah for preview and dev builds. Bad?

Reply:
I don't think there is a way around this. @hishitetsu

Reply:
How about having both an alias that specifies a build and an alias that does not?

Reply:
I wasn't aware of that option. Good solution!

Reply:
Indeed. Let me add a static one

Reply:
Btw, this PR was actually to cope with an issue where the alias is overridden when another Files build installed. So that script, if any, would result in malfunction and thus I don't think letting script writers to use ambiguous alias.

Reply:
@yaira2

Reply:
This is for backwards compatibility.